code/game/objects/items/devices/walkman.dm

```diff
@@ -341,6 +342,7 @@
 	desc = "A plastic cassette tape with a blue sticker."
 	icon_state = "cassette_blue"
 	side1_icon = "cassette_blue"
+	flags_obj = OBJ_IS_HELMET_GARB
```

Comment:
put this on `/obj/item/device/cassette_tape` instead of changing every cassette tape individually